Local.md

```diff
@@ -1,94 +1,121 @@
-* RayCast
-* Endpoint Management
-
-### Terminal
-* iTerm2
-* Zsh
-* Oh-My-Zsh
-* Powerline
-
-Plugins:
-* git
-* Ruby + Rails
-* autojump
-* Duplicate tab on iTerm2
-
-https://towardsdatascience.com/the-ultimate-guide-to-your-terminal-makeover-e11f9b87ac99
-
-
-### Development Environment Setup
-* Xcode
-* Ruby
-* Redis
-* Postgres App
 
+Items marked with ':heavy_dollar_sign:' are paid products or services.
 
-### Programming Tools
-* GitHub Desktop
-* Postico
-* Chrome
+Items marked with ':balance_scale:' are products or services that have both free and paid versions.
 
 
-### Code Editor
-* Atom / SublimeText /  VSCode
-* Plugins: 
-* TabNine
-* RuboCop Autocorrect
-
-### Communication
-* Slack
-* Apple Calendar
-* Fantastical (paid calendar alternative)
-* Apple Mail - Archive when done. Aim to have inbox zero. Unsubscribe from unwanted newsletters and marketing emails.
-* Superhuman (paid mail alternative)
-* Loom
-* Around
-* Zoom
-
-
-### Organisation
-* Apple Notes App
-* Bear (paid notes app alternative)
-* Apple Reminders App
-* 1Password
-* Swish
-
-### Online Tools
-* Google Docs
-* Google Slides
-* Google Sheets
-* Dropbox Paper
-* Figma
-* Miro
-* Jira
-* Linear
-* GitHub
-
+## Initial Setup ☁️
+* Setup Mac with your Apple account
+* Enable FindMyMac
+* Hardware encryption with backup key stored in iCloud
 
-### Mobile Apps to Install
-* Work Calendar 
-* Work Email
-* Authenticator app for 2FA
+|Name|Details|
+|--|--|
+|[Safari](https://www.apple.com/safari/)|Faster speed and more privacy that alternatives| 
+|[RayCast](https://raycast.com)| Powerful alternative to Spotlight|
+|[Endpoint Management](https://go.jamfnow.com/)|Enroll the computer to our company device management system.|
+
+
+## Terminal :computer:
+
+A useful tutorial: https://towardsdatascience.com/the-ultimate-guide-to-your-terminal-makeover-e11f9b87ac99
+
+|Name|Details|
+|--|--|
+|[iTerm2](https://iterm2.com)|Native terminal replacement that allows you do much more| 
+|[Zsh](https://github.com/ohmyzsh/ohmyzsh/wiki/Installing-ZSH)|Command line shell|
+|[Oh-My-Zsh](https://github.com/ohmyzsh/ohmyzsh)|Allows you add amazing plugins and themes to zsh|
+
+
+#### :electric_plug:  [**Plugins**](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins)
+After you set up Oh-My-Zsh, you will have to edit the hidden file './zshrc' to change themes and add useful plugins. Type 'open ./zshrc' in your terminal to find and edit it. These are the plugins to install.
+
+* `git`
+* `Ruby`
+* `Rails`
+* `autojump`
+
+## Development Environment 
+
+|Name|Details
+|--|--|
+|[Xcode](https://developer.apple.com/xcode/)|| 
+|[Ruby](https://www.ruby-lang.org/en/)|A dynamic, open source programming language with a focus on simplicity and productivity|
+|[Rails](https://rubyonrails.org)| A web application development framework for Ruby|
+|[Redis](https://redis.io/download)|Used for cache, jobs queue|
+|[Postgres](https://postgresapp.com)|Our primary SQL database for record storage|
```

Comment:
link all of these to how to install tutorials for Mac?

Reply:
except maybe the Postgres.app. Also give instructions on how they should setup. The idea of this document is that someone can go through this document from top to bottom and by the end of it they will have the exact local setup as we want them to have.